packages/discord.js/src/util/BitField.js

```diff
@@ -140,7 +140,7 @@ class BitField {
 
   *[Symbol.iterator](...hasParams) {
     for (const bitName of Object.keys(this.constructor.Flags)) {
-      if (this.has(bitName, ...hasParams)) yield bitName;
+      if (typeof bitName === 'string' && this.has(bitName, ...hasParams)) yield bitName;
```

Comment:
I don't see how this fixes anything, considering the keys will always be strings 
![Code - Insiders - 2023-04-09 at 18 57 24@2x](https://user-images.githubusercontent.com/17960496/230783250-26a8145f-c9e2-46ec-a4ab-8d0ce264cabc.png)

Reply:
Oh sorry if forgot to change the commit 
Now its working fine 
![image](https://user-images.githubusercontent.com/76836550/230784006-ef48fea9-e82e-4b81-b31c-e4dd4f7a4e46.png)